docs/line.asciidoc

```diff
@@ -27,9 +27,27 @@ NOTE: In Elasticsearch releases 1.4.3 and later, this functionality requires you
 {ref}/modules-scripting.html[dynamic Groovy scripting].
 
 The availability of these options varies depending on the aggregation you choose.
-Select *view options* to change the following aspects of the table:
 
+Select the *Options* tab to change the following aspects of the chart:
+
+*Y-Axis Scale*:: You can select *linear*, *log*, or *square root* scales for the chart's Y axis.
+*Smooth Lines*:: Check this box to curve the line from point to point.
```

Comment:
Added the sentence " Bear in mind that smoothed lines necessarily affect the representation of your data and create a potential for ambiguity."